time Stamp
float64 Distance
